### ISSUE TITLE
Unable to swipe to table cells containing the status-dot component when using VoiceOver/Safari/iOS 12.1.4

### ISSUE THREAD

**Issue by chlane** [vmware/clarity#3183]
## Description
The user is unable to swipe to table cells containing the status-dot component when using VoiceOver/Safari/iOS 12.1.4 on the Component Status table on https://clarity.design/documentation. An example of one of these table cells is below.

`<td><status-dot><img src="assets/images/documentation/status/done.svg" title="Work completed" alt="Work completed"></status-dot></td>`

## Location
https://clarity.design/documentation

## Priority
High

## Recommended fix
Unknown

## LOE
High

**Comment by gnomeontherun** [comment]
No longer an issue or resolved in the upcoming website work. Closing as resolved.
